src/patternfly/components/Wizard/wizard.scss

```diff
@@ -441,7 +441,9 @@
   word-break: break-word;
 
   &.pf-m-no-padding {
-    padding: 0;
+    .pf-c-wizard__main-body {
```

Comment:
Why did you put the modifier on `.pf-c-wizard__main` instead of `.pf-c-wizard__main-body`? Is it because of breaking changes? Maybe create a follow up issue?